### DISCUSSION TITLE
Monocular to blender

### DISCUSSION THREAD

**Discussion by marifroe** [zju3dv/EasyMocap#199] [Q&A] [answered]
Hi everyone,

first of all: sorry, I'm pretty new to this and still a bit confused. Actually, started looking into motion capture and easymocap beginning of this week. So far, after facing some challenges, I successfully set everything up. I extracted keypoints (running scripts/preprocess/extract_video.py using openpose) and smpl data (running apps/demo/mocap.py) and am now trying to import the output into blender. Therefore, I am running scripts/postprocess/convert2bvh.py which says "Import finished. Success load" but doesn't give me any output. What am I missing here? In the first place, is it supposed to work like this? Sorry again if this is a dumb question. I would really appreciate your help!

Thanks, Marilena

**Comment by funnyQZQ** [comment]
Maybe, you should check the scripts/postprocess/convert2bvh.py pipeline, i have converted the output smpl pose to bvh file and import it to blender successfully.

**Reply by marifroe** [reply]
Thanks for your reply! Was good to know that it should actually work.

**Comment by marifroe** [comment, marked as answer]
Got it running. I was using the wrong path to smpl data. Works fine with path to smplfull. Also, there was a typo in the script in line 155 with a n missing in 'isinsta**n**ce'.
